client/src/App.js

```diff
@@ -90,23 +104,111 @@ function App() {
   const handleRankSubmit = (e) => {
     e.preventDefault();
     console.log(rankValue);
-  };
+  }; */
   return (
-    //--- Selection Bar,Show level, amount of AGRD
-    <Fragment>
-      <div class="container">
-        <div class="container">
-          <div class="row">
-            <div class="col">AREA FOR BUTTONS</div>
-            <div>{userAddress ? userAddress : ""}</div>
-            <div class="col-md-auto">
-              Your Tier: {rank ? UserRank[rank] : "None"}
-            </div>
-            <div class="col col-lg-2">
-              Your Balance: {balance ? balance : ""}
-            </div>
-
-            <div class="col col-lg-2">
+    <Container fluid style={{ backgroundColor: "#a1fcd9", minHeight: "100vh" }}>
+      {/* Navbar  */}
+      <Row>
+        <Navbar collapseOnSelect expand="md" bg="light" variant="light">
+          <Container fluid>
+            <img src="./logo.png" width="70" height="70" alt="logo" />
+            <Navbar.Brand href="#home">
+              <h3>Contracts AGREED Upon </h3>
+            </Navbar.Brand>
+            <Navbar.Toggle aria-controls="responsive-navbar-nav" />
+
+            <Navbar.Collapse id="responsive-navbar-nav">
+              <Nav className="container-fluid">
+                <Nav.Item className="ms-5">
+                  <h5>
+                    {userAddress
+                      ? "Wallet: " + userAddress.substring(0, 7) + "..."
+                      : ""}
+                  </h5>
+                </Nav.Item>
+                <Nav.Item className="ms-auto">
+                  <h5>{rank >= 0 ? "LVL: " + UserRank[rank] : ""}</h5>
+                </Nav.Item>
+                <Nav.Item className="ms-5">
+                  <h5>{balance ? "AGRD: " + balance : ""}</h5>
+                </Nav.Item>
+              </Nav>
+            </Navbar.Collapse>
+          </Container>
+        </Navbar>
+      </Row>
+      {/* Main Content */}
+      <Row style={{ marginTop: "10px", marginBottom: "10px" }}>
+        {/* left side */}
+        <Col xs={12} md={8}>
+          <Container>
+            {/* Cards */}
+            <Row className="flex-grow-1 justify-content-center align-items-center">
+              {true /* change to these values for real scenario:
+                     rank
+                      ? Array(rank) 
+                      
+                      for testing purpose it shows all 4 possible ranks*/
+                ? Array(4)
```

Comment:
Only question and then I'll merge, so here am I changing line 152 to Array(rank) right??

Reply:
That was my assumption, thanks for the clarification!